connectors/operate/element-templates/operate-connector.json

```diff
@@ -0,0 +1,506 @@
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "input.internal_filter"
+      },
+      "optional": true,
+      "condition": {
+        "property": "input.internal_operation",
+        "equals": "/search"
+      }
+    },
+    {
+      "label": "Sort",
+      "description": "Sorting properties in <a href=\"https://docs.camunda.io/docs/apis-clients/operate-api/#sort\" target=\"_blank\">Operate format</a><br>Please provide a list of sort objects",
+      "group": "parameters",
+      "type": "String",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "input.internal_sort"
+      },
+      "optional": true,
+      "condition": {
+        "property": "input.internal_operation",
+        "equals": "/search"
+      }
+    },
+    {
+      "label": "Results",
+      "description": "Number of results to return",
+      "group": "parameters",
+      "type": "String",
+      "value": "20",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "input.internal_results"
+      },
+      "condition": {
+        "property": "input.internal_operation",
+        "equals": "/search"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "pattern": {
+          "value": "^[1-9]\\d*$",
+          "message": "Must be a positive integer number"
+        }
+      }
+    },
+    {
+      "label": "Pagination",
+      "description": "Identifier of item from which the search should start. <br>Hint: Copy the <code>sortValues</code> field from the previous search result, or leave blank for no pagination",
+      "group": "parameters",
+      "type": "String",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "input.internal_pagination"
+      },
+      "optional": true,
+      "condition": {
+        "property": "input.internal_operation",
+        "equals": "/search"
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "={filter: if is defined(input.internal_filter) then input.internal_filter else null, sort: if is defined(input.internal_sort) then input.internal_sort else null, searchAfter: if is defined(input.internal_pagination) then input.internal_pagination else null, size: number(input.internal_results)}",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body"
+      },
+      "condition": {
+        "property": "input.internal_operation",
+        "equals": "/search"
+      }
+    },
+    {
+      "label": "Result Variable",
+      "description": "Name of variable to store the response in. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/use-connectors/#result-variable\" target=\"_blank\">documentation</a>",
+      "group": "output",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:taskHeader",
+        "key": "resultVariable"
+      }
+    },
+    {
+      "label": "Result Expression",
```

Comment:
Just a thought: Does it make sense to add predefined values in the Result Expression for each operations as a suggestion or for convenience?